scripts/us_census/pep/pep_by_asrh/preprocess_test.py

```diff
@@ -0,0 +1,102 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Script to automate the testing for USA Population preprocess script.
```

Comment:
The file cols_map.py has a bunch of dictionaries which are abstracted/received by calling individual functions. I recommend have one function which can return them all as a dictionary/map and then having a test here to ensure that every single dictionary created in cols_map.py is also used in the preprocess.py.